services/portal/GWASUIApp/GWASUIAppProps.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+    path: 'analysis/GWASUIApp',
+    resultsPath: 'analysis/GWASResults',
+
+    // GWAS++ elements
+    NextSpan: '//span[contains(text(),\'Next\')]',
+    PreviousSpan: '//span[contains(text(),\'Previous\')]',
+    GWASWindow: '//div[@class="select-container"]',
+    checkedRadio: '(//*[@class="ant-radio ant-radio-checked"])',
+    // from step1
+    AddCohortButton: { css: 'button[data-tour="cohort-add"]' },
```

Comment:
@jarvisraymond-uchicago  I have used the css selector here for the `Add cohort` button. It will work until someone changes the id of the button.